src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -76,11 +76,14 @@ public Object processRequest(Request request, Response response, String userId)
                 Patch patch = gson.fromJson(requestBody, Patch.class);
                 if (StringUtils.isNotBlank(patch.getId())) {
                     //multiple values are changing
+                    DDPInstance ddpInstance = DDPInstance.getDDPInstance(patch.getRealm());
                     if (patch.getNameValues() != null && !patch.getNameValues().isEmpty()) {
                         List<NameValue> nameValues = new ArrayList<>();
-                        DDPInstance ddpInstance = DDPInstance.getDDPInstance(patch.getRealm());
-                        ESProfile profile = ElasticSearchUtil.getParticipantProfileByGuidOrAltPid(ddpInstance.getParticipantIndexES(), patch.getParentId())
-                                .orElseThrow(() -> new RuntimeException("Unable to find ES profile for participant: " + patch.getParentId()));
+                        ESProfile profile = ElasticSearchUtil.getParticipantProfileByGuidOrAltPid(ddpInstance.getParticipantIndexES(), patch.getDdpParticipantId())
+                                .orElse(null);
+                        if (profile == null) {
+                            logger.error("Unable to find ES profile for participant with guid/altpid: {}, continuing w/ patch", patch.getParentId());
```

Comment:
removed the throw and logging error now, because we still want to patch the value in the db even if we do not write into ES